datasets/ecmwf-forecast/collection/ecmwf-forecast-enfo-ep/template.json

```diff
@@ -0,0 +1,101 @@
+    "item_assets": {
```

Comment:
Just remembered, we'll want to update this to ensure that we get efficient compression on the STAC items in the database. Can chat more, but we want to include as much of `kerhcunk:indices` as possible here, everything that's static across all items in the dataset. Hopefully stuff like the `.zattrs` and `.zarray` are static, but maybe not.

Reply:
Done.